selfdrive/car/fw_versions.py

```diff
@@ -203,19 +204,30 @@ def get_fw_versions_ordered(logcan, sendcan, ecu_rx_addrs, timeout=0.1, num_pand
 
   all_car_fw = []
   brand_matches = get_brand_ecu_matches(ecu_rx_addrs)
+  matched_brand: Optional[str] = None
 
   for brand in sorted(brand_matches, key=lambda b: len(brand_matches[b]), reverse=True):
     car_fw = get_fw_versions(logcan, sendcan, query_brand=brand, timeout=timeout, num_pandas=num_pandas, debug=debug, progress=progress)
     all_car_fw.extend(car_fw)
     # Try to match using FW returned from this brand only
     matches = match_fw_to_car_exact(build_fw_dict(car_fw))
     if len(matches) == 1:
+      matched_brand = brand
       break
 
+  # Do non-OBD queries for matched brand, or all if no match is found
+  params = Params()
+  params.put_bool("FirmwareObdQueryDone", True)
+  params.get_bool("ObdMultiplexingDisabled", block=True)
+
+  for brand in FW_QUERY_CONFIGS.keys():
+    if brand == matched_brand or matched_brand is None:
```

Comment:
why do it if we don't have a match?

Reply:
If the Honda Bosch fuzzy matches, I'd like to still query it

Reply:
I think we might get less data, but not required. I have a suspicion a lot of Hondas fuzzy FP because they have so many ECUs

Reply:
let's just check the data. this is cloudlogged, so you can just check kibana quickly

Reply:
Last 90 days:

```python
169 dongles are mostly fuzzy FP, out of 208 total dongles
15 platform are mostly fuzzy FP, out of 17 total platforms
```